docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -81,6 +81,31 @@ The path to the Teku data directory. The default directory is OS dependent:
 
 The default Docker image location is `/root/.local/share/teku`.
 
+### data-storage-archive-frequency
+
+```bash tab="Syntax"
+--data-storage-archive-frequency=<NUMBER>
+```
+
+```bash tab="Command Line"
+--data-storage-archive-frequency=1028
+```
+
+```bash tab="Environment Variable"
+TEKU_DATA_STORAGE_ARCHIVE_FREQUENCY=1028
+```
+
+```bash tab="Configuration File"
+data-storage-archive-frequency: 1028
+```
+
+Set the frequency (in slots) at which to store finalized states to disk. Defaults to 2048.
+
+This option is ignored if [`--data-storage-mode`](#data-storage-mode) is set to `prune`.
+
+!!! note
+    The lower the frequency of stored states, the higher the overhead of retrieving a state from disk.
```

Comment:
so it's more that the overhead increases as there are more states that potentially need to be regenerated in order to get to the state requested...
ie. if it's 2048, you may need to get a state 2047 slots behind the one you request, and then replay blocks to get up to the one you've requested, which is time consuming. so as the storage frequency increases, the number of blocks to replay decreases, which will make state retrieval 'faster'.

i guess you could term it `the higher the overhead of retrieving a finalized state`, dropping `from disk`.

On the flip side, increasing the number of stored states will increase disk usage, so it's a trade off.